### ISSUE TITLE
additional para for

### ISSUE THREAD

**Issue by yiluheihei** [joey711/phyloseq#1053]

**Comment by joey711** [comment]
Hi @yiluheihei feel free to write a pull request. Before you do, you may want to search around. I think there may be other drafts/prototypes laying around from other contributors. 

Before I do so, and change the title, do I understand correctly that you are asking for a functionality that mimics the qiime `summarize_taxa` method? I'm asking to clarify because that is not the title of the ticket, though it is mostly the content.

**Comment by yiluheihei** [comment]
@joey711 Yes, just a functionality that mimics the qiime `summarize_taxa` method.
